src/feature/hs/hs_descriptor.c

```diff
@@ -240,53 +240,73 @@ build_mac(const uint8_t *mac_key, size_t mac_key_len,
   crypto_digest_free(digest);
 }
 
-/* Using a given decriptor object, build the secret input needed for the
- * KDF and put it in the dst pointer which is an already allocated buffer
- * of size dstlen. */
-static void
-build_secret_input(const hs_descriptor_t *desc, uint8_t *dst, size_t dstlen)
+/* Using a secret data and a given decriptor object, build the secret
```

Comment:
FYI, this comes up a lot: When documenting functions in the future, please use "/**", not "/*".  The first format tells doxygen to parse the comment.

But also please don't make this change on this branch -- I bet it would cause a lot of conflicts.

Reply:
Hmmm the entire HSv3 subsystem uses `/*` unfortunately... So we could do one commit to fix all of those in the `hs/` directory :).

Reply:
Yeah, I also have seen `/*` in many places :)